homeassistant/components/tts/__init__.py

```diff
@@ -124,6 +125,7 @@ def valid_base_url(value: str) -> str:
         vol.Optional(ATTR_CACHE): cv.boolean,
         vol.Required(ATTR_ENTITY_ID): cv.comp_entity_ids,
         vol.Optional(ATTR_LANGUAGE): cv.string,
+        vol.Optional(ATTR_MEDIA_EXTRA): dict,
```

Comment:
I personally don't like unmanaged structures. It solve the use case of YAML but is against streamlining or making it accessible from UI. But that is just my comment on that

Reply:
It's not very pretty, but it's defined like this in media_player: https://github.com/home-assistant/core/blob/80653463bfcbe29410c95f77f3ae0ceba3c067e8/homeassistant/components/media_player/__init__.py#L173

Reply:
Let's make sure we keep it in sync
```suggestion
        vol.Optional(ATTR_MEDIA_EXTRA): MEDIA_PLAYER_PLAY_MEDIA_SCHEMA[ATTR_MEDIA_EXTRA],
```

Reply:
Oh, that's neat 👍